apps/chat/src/components/Chatbar/ModelIcon.tsx

```diff
@@ -72,8 +74,9 @@ const ModelIconTemplate = memo(
           width={size}
           height={size}
           onError={handleError}
-          alt={description}
+          data-qa={description}
```

Comment:
It would be much better to use here something else, not the data-qa=something. It will lead to undesirable consequences.  We can introduce a new exclusive attribute.